packager/templates/debian/global/choria-logrotate

```diff
@@ -1,4 +1,4 @@
-/var/log/{{cpkg_name}}-audit.log
```

Comment:
@smortex if you do do logrotation setup this is going to be the safest bet as things like the old mco log will fit into this pattern